test/e2e/forwarded_header_policy_test.go

```diff
@@ -121,7 +121,8 @@ func testRouteHeaders(t *testing.T, image string, route *routev1.Route, address
 // client specifies 2 X-Forwarded-For headers, then the router should append a
 // 3rd.
 func TestForwardedHeaderPolicyAppend(t *testing.T) {
-	icName := types.NamespacedName{Namespace: operatorNamespace, Name: "forwardedheader"}
+	t.Parallel()
+	icName := types.NamespacedName{Namespace: operatorNamespace, Name: "forwardedheader-append"}
```

Comment:
I believe `testRouteHeaders` will be problematic; we need something like this change to prevent conflicts:
```diff
diff --git a/test/e2e/forwarded_header_policy_test.go b/test/e2e/forwarded_header_policy_test.go
index a5d5ec9c..b1d5b562 100644
--- a/test/e2e/forwarded_header_policy_test.go
+++ b/test/e2e/forwarded_header_policy_test.go
@@ -52,7 +52,7 @@ func testRouteHeaders(t *testing.T, image string, route *routev1.Route, address
 		extraCurlArgs = append(extraCurlArgs, "-H", header)
 	}
 	testPodCount++
-	name := fmt.Sprintf("forwardedheader%d", testPodCount)
+	name := fmt.Sprintf("%s%d", route.Name, testPodCount)
 	clientPod := buildCurlPod(name, route.Namespace, image, route.Spec.Host, address, extraCurlArgs...)
 	if err := kclient.Create(context.TODO(), clientPod); err != nil {
 		t.Fatalf("failed to create pod %s/%s: %v", clientPod.Namespace, clientPod.Name, err)
```

Reply:
Nice catch! Thanks.

Reply:
Fixed here: 416d29f0dafc8d5d85e8fefecabb4582abb6f784